include/rangers-api/Application/Player/PlayerHsmContext.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+namespace app::player {
+    class PlayerHsmContext : public hh::fnd::ReferencedObject, public app::save::SaveManagerListener {
+        class OutOfControlTimerList : public PlayerCounterTimer {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.